mangaki/mangaki/utils/mal.py

```diff
@@ -345,18 +345,27 @@ def insert_into_mangaki_database_from_mal(mal_entries: List[MALEntry],
                 and title.upper() in list(map(str.upper, titles))):
                 first_matching_work = work
 
+            # FIXME: will create many redundant queries.
+            simple_lang = Language.objects.get(code='simple')
```

Comment:
Why not use a cache like in the AniDB API? Besides, see above comment on keeping `ExtLanguages` -- we should have a couple `ExtLanguage` instances for MAL as well.